README.md

```diff
@@ -12,7 +12,7 @@ You need [Docker](https://docs.docker.com/engine/installation/) and [Rust](https
 
 ```sh
 # Clone the project:
-$ git clone https://github.com/serlo/serlo.org-database-layer.git
+$ git clone https://github.com/serlo/database-layer.git
 $ cd serlo.org-database-layer
```

Comment:
```suggestion
$ cd database-layer
```

Reply:
But, to be honest, I think this block (lines 13 to 17) is unnecessary, because it is obvious (besides that, some prefer cloning using ssh)